src/Elmish.WPF/ViewModel.fs

```diff
@@ -78,7 +77,11 @@ and internal SubModelSelectedItemBinding<'model, 'msg, 'bindingModel, 'bindingMs
   Get: 'model -> 'id voption
   Set: 'id voption -> 'model -> unit
   SubModelSeqBinding: SubModelSeqBinding<'model, 'msg, obj, obj, obj>
-  Selected: Lazy<ViewModel<'bindingModel, 'bindingMsg> voption> ref
+}
+
+and internal CachedBinding<'model, 'msg> = {
+  Binding: VmBinding<'model, 'msg>
+  Cache: obj option ref
```

Comment:
Should we parametrize `CachedBinding` by the `Cache` type, too? And only assign `obj` when used below in `VmBinding`? This would make it consistent with the other bindings.

Reply:
Sure.  I squashed this change into the first commit and force pushed.